libmultilabel/evaluate.py

```diff
@@ -0,0 +1,104 @@
+import logging
+import re
+
+import numpy as np
+from sklearn.metrics import f1_score, precision_recall_fscore_support
+from tqdm import tqdm
+
+from .metrics import another_macro_f1, precision_recall_at_ks
+from .utils import Timer, dump_log, save_top_k_prediction
+
+
+def evaluate(model, dataset_loader, num_classes, monitor_metrics, label_key='label', predict_out_path=None, save_k_predictions=0):
```

Comment:
``num_classes`` is not used in ``MultiLabelMetrics`` anymore, which can be removed